packages/edit-widgets/src/components/keyboard-shortcuts/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+import { useShortcut } from '@wordpress/keyboard-shortcuts';
+import { useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+function KeyboardShortcuts() {
+	const { redo, undo } = useDispatch( 'core' );
+
+	useShortcut(
+		'core/editor/undo',
+		( event ) => {
+			undo();
+			event.preventDefault();
+		},
+		{ bindGlobal: true }
+	);
+
+	useShortcut(
+		'core/editor/redo',
+		( event ) => {
+			redo();
+			event.preventDefault();
+		},
+		{ bindGlobal: true }
+	);
+
+	return null;
+}
+function KeyboardShortcutsRegister() {
+	// Registering the shortcuts
+	const { registerShortcut } = useDispatch( 'core/keyboard-shortcuts' );
+	useEffect( () => {
+		registerShortcut( {
+			name: 'core/editor/undo',
```

Comment:
In edit-post it looks like we are using naming convention of `core/edit-post/undo`, should these be `core/edit-site/undo` / similarly for widgets?

Reply:
Hi @Addison-Stavlo nice catch, I updated the code 👍